object-challenge/challenges-06.test.js

```diff
@@ -0,0 +1,284 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named getCourseKeys that takes in the courseInfo object and returns an array containing the keys for the courseInfo object.
+
+For example: (['name', 'duration', 'topics', 'finalExam']).
+------------------------------------------------------------------------------------------------ */
+
+const courseInfo = { name: 'Code 301', duration: { dayTrack: '4 weeks', eveningTrack: '8 weeks'},
+  topics: ['SMACSS', 'APIs', 'NodeJS', 'SQL', 'jQuery', 'functional programming'],
+  finalExam: true };
+
+const getCourseKeys = (obj) => {
+  // Solution code here...
+  const keys = Object.keys(obj);
```

Comment:
`Object.keys()` will return an array containing the keys of the object passed in. You could have just returns keys!